package.json

```diff
@@ -2,6 +2,7 @@
   "name": "android-emulator-webrtc",
   "version": "1.0.18",
   "description": "Android Emulator WebRTC module",
+  "types": "emulator/index.d.ts",
```

Comment:
This doesn't make sense; nothing about this package.json implies importing it will point to this file, which is what this line is saying.

Reply:
Sorry for the late reply, but thanks for the review! You're right—I mixed up my local declarations while porting here. I've fixed and tested that it works as expected now.